bitnami/redis/templates/master/statefulset.yaml

```diff
@@ -276,6 +276,11 @@ spec:
               mountPath: /opt/bitnami/redis/certs
               readOnly: true
             {{- end }}
+            {{- if .Values.metrics.extraArgs.script }}
```

Comment:
I don't necessarily trust the `redis_exporter` chart to do the right thing with a `script: null` parameter, but I know for a fact it will behave correctly if the key is not set.

I can try to scope this to just the `metrics` container, but the general idea was to leverage the existing `extraVolumeMounts` support to keep it simple; adding another `extraVolumeMounts` code path just for `metrics` might make this change more complex than it's worth.

Reply:
Thanks for the context. 

> I don't necessarily trust the `redis_exporter` chart to do the right thing with a `script: null` parameter

You can always manage these exceptions but I guarantee you that you won't find any issue if the value exists and it's set to `null`. 

> but the general idea was to leverage the existing extraVolumeMounts support to keep it simple; adding another extraVolumeMounts code path just for metrics might make this change more complex than it's worth.

I agree with keeping the chart simple. That's why I'd rather stick to the existing parameters (`extraVolumeMounts`/`extraVolumes`) since adding new parameters actually increase the maintenance costs.

I think it's just a matter of having  good docs describing how to achieve your use case with the existing parameters.

Reply:
OK, this has been re-homed to https://github.com/bitnami/charts-docs/pull/56

Reply:
I agree with @juan131, @0xDEC0DE. I think we should stick to `metrics.extraVolumeMounts`. We already have a `extraVolumeMounts` parameter for the master, the replica, and the sentinel, so it makes sense to have another one for the metrics. It is consistent and flexible, as you can mount whatever you want there and not just a script.

We can also add a documentation page like the one you just created explaining how to setup a custom script using this new `metrics.extraVolumeMounts` parameter if you feel like it is necessary.

Reply:
Done.